src/components/Dashboard/SortList/storage.js

```diff
@@ -0,0 +1,16 @@
+const storage = {
+  get: (key) => {
+    const storageKey = `SortList_Data_${(key || 'null').toString().toLowerCase().replace(/[^0-9a-z]/ig, '')}`;
```

Comment:
If I understood correctly from the code above, the `key` variable is already either something established or a `'null'` string (which is very weird on itself, but that's not the point right now). So, why do you need the logical `||` operator since the `key` variable will never yield a falsy value?

Reply:
Fixed